packages/0xcert-scaffold/src/core/messages.ts

```diff
@@ -0,0 +1,91 @@
+  [ProviderIssue.NO_RECEIVER_ID]: '`receiverId` is not set.',
+  [ProviderIssue.SENDER_ID_NOT_A_SIGNER]: 'SenderId not a signer.',
+  [ProviderIssue.SENDER_ID_AND_RECEIVER_ID_MISSING]: 'Both senderId and receiverId missing.',
+  [ProviderIssue.ACTION_KIND_NOT_SUPPORTED]: 'Not implemented.',
+  [ProviderIssue.ERC20_APPROVAL_RACE_CONDITION]: 'First set approval to 0. ERC-20 token potential attack.',
+  [ProviderIssue.ERC20_NOT_ENOUGH_BALANCE]: 'Sender does not have enough balance.',
+  [ProviderIssue.ERC20_NOT_ENOUGH_ALLOWANCE]: 'You do not have enough allowance.',
+  [ProviderIssue.ERC721_ZERO_ADDRESS]: 'Provided address cannot be zero address.',
+  [ProviderIssue.ERC721_NOT_VALID_NFT]: 'Asset with this id does not exist.',
+  [ProviderIssue.ERC721_NOT_OWNER_OR_OPERATOR]: 'Sender is neither asset owner or operator.',
+  [ProviderIssue.ERC721_NOT_OWNER_APPROWED_OR_OPERATOR]: 'Sender is neither asset owner, approved or operator.',
+  [ProviderIssue.ERC721_NOT_ABLE_TO_RECEIVE_NFT]: 'Receiver is not able to safely receive the asset.',
+  [ProviderIssue.ERC721_NFT_ALREADY_EXISTS]: 'Asset with provided id already exists.',
+  [ProviderIssue.ERC721_METADATA_ZERO_ADDRESS]: 'Provided address cannot be zero address.',
+  [ProviderIssue.ERC721_METADATA_NOT_VALID_NFT]: 'Asset with this id does not exist.',
+  [ProviderIssue.ERC721_METADATA_NOT_OWNER_OR_OPERATOR]: 'Sender is neither asset owner or operator.',
+  [ProviderIssue.ERC721_METADATA_NOT_OWNER_APPROWED_OR_OPERATOR]: 'Sender is neither asset owner, approved or operator.',
+  [ProviderIssue.ERC721_METADATA_NOT_ABLE_TO_RECEIVE_NFT]: 'Receiver is not able to safely receive the asset.',
+  [ProviderIssue.ERC721_METADATA_NFT_ALREADY_EXISTS]: 'Asset with provided id already exists.',
+  [ProviderIssue.ERC721_ENUMERABLE_ZERO_ADDRESS]: 'Provided address cannot be zero address.',
+  [ProviderIssue.ERC721_ENUMERABLE_NOT_VALID_NFT]: 'Asset with this id does not exist.',
+  [ProviderIssue.ERC721_ENUMERABLE_NOT_OWNER_OR_OPERATOR]: 'Sender is neither asset owner or operator.',
+  [ProviderIssue.ERC721_ENUMERABLE_NOT_OWNER_APPROWED_OR_OPERATOR]: 'Sender is neither asset owner, approved or operator.',
+  [ProviderIssue.ERC721_ENUMERABLE_NOT_ABLE_TO_RECEIVE_NFT]: 'Receiver is not able to safely receive the asset.',
+  [ProviderIssue.ERC721_ENUMERABLE_NFT_ALREADY_EXISTS]: 'Asset with provided id already exists.',
+  [ProviderIssue.ERC721_ENUMERABLE_INVALID_INDEX]: 'There is no asset at provided index.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_ZERO_ADDRESS]: 'Provided address cannot be zero address.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_NOT_VALID_NFT]: 'Asset with this id does not exist.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_NOT_OWNER_OR_OPERATOR]: 'Sender is neither asset owner or operator.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_NOT_OWNER_APPROWED_OR_OPERATOR]: 'Sender is neither asset owner, approved or operator.',
```

Comment:
[ProviderIssue.ERC721_METADATA_ENUMERABLE_NOT_OWNER_APPROWED_NOR_OPERATOR]: 'Sender is neither asset owner, approved nor operator.',